gh-emoji.js

```diff
@@ -13,19 +17,31 @@ export function load() {
   });
 }
 
-export function all() {
+export function all(): ?EmojiMap {
   return emojis;
 }
 
-export function exist(emojiId) {
-  return !!all()[emojiId];
+export function exist(emojiId: string): boolean {
+  const emojis = all(); // eslint-disable-line no-shadow
```

Comment:
I don't think that we should disable eslint rules for specific blocks / lines, imo we should change the name of our global emojis variable to emojiMap or smth. What do you think?